apps/server/src/modules/authorization/errors/forbidden.loggable-exception.ts

```diff
@@ -0,0 +1,26 @@
+import { ForbiddenException } from '@nestjs/common';
+import { EntityId, IPermissionContext } from '@shared/domain';
+import { Loggable } from '@src/core/logger/interfaces';
+import { ErrorLogMessage } from '@src/core/logger/types';
+
+export class ForbiddenLoggableException extends ForbiddenException implements Loggable {
+	constructor(private userId: EntityId, private context: IPermissionContext) {
+		super();
+	}
+
+	getLogMessage(): ErrorLogMessage {
+		const message = {
+			type: 'FORBIDDEN_EXCEPTION',
+			stack: this.stack,
+			data: {
+				userId: this.userId,
+				// TODO: This is not easily possible atm because of the PermissionTypes, where id is optional in BaseDO.
+				// entityId: this.entityId,
```

Comment:
entityId als optionalen Parameter in den constructor geben ist hier keine Option?

Reply:
Auch hier sollten wir nochmal drüber reden, was da eigentlich in data rein soll. Das hab ich einfach mal so vorläufig entschieden, wie es mir sinnvoll erschien und orientiert an dem Entwurf von Thomas. Der Kommentar ist dann zugegeben etwas verwirrend.